### PR TITLE
Add command modul

### PR DESCRIPTION
**Pull request by tim-boerner**

### CODE REVIEW

File: Kitodo-Command/pom.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tim-boerner) [inline Kitodo-Command/pom.xml]: done

File: Kitodo-Command/src/main/java/org/kitodo/command/Command.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package org.kitodo.command;
```
Comment (tim-boerner) [inline Kitodo-Command/src/main/java/org/kitodo/command/Command.java]: done

File: Kitodo-Command/src/main/java/org/kitodo/command/Command.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.kitodo.api.command.CommandInterface;
+import org.kitodo.api.command.CommandResult;
+
+public class Command implements CommandInterface {
+
+    @Override
+    public CommandResult runCommand(Integer id, String command) {
+        return null;
```
Comment (henning-gerhardt) [inline Kitodo-Command/src/main/java/org/kitodo/command/Command.java]: Returning `null` should be avoided even the implementation is not done yet. In my opinion I see two solutions:
- return a `CommandResult` object which indicate that the implementation is not done
- throw a RuntimeException with a message that the implementation is missing.

Comment (Kathrin-Huber) [inline Kitodo-Command/src/main/java/org/kitodo/command/Command.java]: throwing a runtime exception would require an adjustment of the api, so i would not recommend that ;)

Comment (henning-gerhardt) [inline Kitodo-Command/src/main/java/org/kitodo/command/Command.java]: Even for the implementation phase?

Comment (Kathrin-Huber) [inline Kitodo-Command/src/main/java/org/kitodo/command/Command.java]: I just realised, that RuntimeException is unchecked, so the signature can stay as it is and everything is fine. :)

**Pull request merged**